src/util/serialization/ast.cpp

```diff
@@ -142,6 +143,139 @@ namespace phylanx { namespace util
         detail::unserialize_helper(input, expr);
         return expr;
     }
-}}
 
+    // You cannot add to expression.rest unless you
+    // use this function. Note sure why.
+    void append_operation(ast::expression& ast,ast::operation const& o) {
+      ast.rest.push_back(o);
+    }
```

Comment:
`expression has a member `append()` just for this (see: https://github.com/STEllAR-GROUP/phylanx/blob/master/phylanx/ast/node.hpp#L472-L487)

Reply:
OK, I thought that was the built-in pybind11 command for lists. It didn't work for me. Calling it left the list empty.

Reply:
Ok, good point - I'll have a look and fix this. Thanks!